dynamic_programming/min_insertions_deletions.py

```diff
@@ -0,0 +1,115 @@
+"""
+Problem Statement :
+
+Given two strings s1 & s2 of length m and n respectively,
+transform s1 to s2 with minimum number of insertions and deletions
+
+Algorithm :
+Idea is to use classic DP algorithm, LCS(Longest Common Subsequence)
+Example, given str1 = "foam" and str2 = "programming"
+
+LCS(str1, str2) = L = "oam"  => length(L) = 3
+Thus,
+deletions = len(str1) - len(L), 
+insertions = len(str2) - len(L)
+"""
+
+
+def longest_common_subsequence(str1: str, str2: str):
```

Comment:
```suggestion
def longest_common_subsequence(str1: str, str2: str) -> int:
```

add type hinting for return value